crates/sparrow-session/src/table.rs

```diff
@@ -134,7 +137,7 @@ impl Table {
         Ok(())
     }
 
-    pub async fn add_parquet(&self, path: &std::path::Path) -> error_stack::Result<(), Error> {
+    pub async fn add_parquet(&self, path: &str) -> error_stack::Result<(), Error> {
```

Comment:
Should this take `&ObjectStoreUrl`?

Reply:
Most likely? But for now, the prepare code still uses `SourceData` which uses a string and stuff, so I can change that once we get rid of that code.

Reply:
Ah. Hmm... seems like that could be a good one to work on soon (relatively self contained).

Reply:
I'll add a task for it